__tests__/__utils__/index.ts

```diff
@@ -23,3 +23,12 @@
 export * from './expect-helper'
 export * from './kv'
 export * from './services'
+
+export enum Backend {
+  Frontend = 'frontend',
+  Legacy = 'legacy',
+}
+
+export function setupProbabilityFor(backend: Backend) {
+  global.FRONTEND_PROBABILITY = backend === Backend.Frontend ? '1' : '0'
+}
```

Comment:
I noticed that when a test suite like `maintenance.ts` imports from `frontend-proxy.ts` the tests from `frontend-proxy.ts` are executed twice. I'll do a follow up PR which cleans up the utility functions.